controller/engine_controller.go

```diff
@@ -352,7 +360,9 @@ func (ec *EngineController) CreatePodSpec(obj interface{}) (*v1.Pod, error) {
 		"--listen", "0.0.0.0:" + engineapi.ControllerDefaultPort,
 		"--size", strconv.FormatInt(e.Spec.VolumeSize, 10),
 	}
-	if frontend != "" {
+	if frontend == "" || e.Spec.DisableFrontend {
+		cmd = append(cmd, "--frontend", "")
```

Comment:
I don't think this will work. It's likely the parameter after `--frontend` will be treated as the frontend name then errors out.